### PR TITLE
Link to BURF format

### PR DESCRIPTION
**Pull request by vturpin**
I change the wording regarding OG1.0
I've added the link to the Ocean Glider's BUFR format

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -9,7 +9,8 @@ A good start is also to [introduce you](https://github.com/OceanGlidersCommunity
 # Ongoing community efforts
 - [GitHub training](https://github.com/OceanGlidersCommunity/LearningGitHub/discussions/53) 🤓🎖
 - [Meeting notes](https://github.com/OceanGlidersCommunity/meeting_notes)
-- Access and discuss the new [OceanGliders format](https://github.com/OceanGlidersCommunity/OG1.0-user-manual).
+- Access and discuss the [OceanGliders format] - OG1.0 (https://github.com/OceanGlidersCommunity/OG1.0-user-manual).
+- Ocean Glider’s BUFR format, check the new sequence here: https://github.com/wmo-im/BUFR4/issues/16
```
Comment (castelao) [inline profile/README.md]: Just a suggestion:

"- Ocean Glider’s BUFR format defined in [WMO's table D by descriptor 3-15-012](https://github.com/wmo-im/BUFR4/blob/master/BUFR_TableD_en_15.csv). The archived proposal and related discussions can be found at: https://github.com/wmo-im/BUFR4/issues/16"

Comment (vturpin) [reply profile/README.md]: I've just made the modifications.

**Review by castelao — APPROVED** [review]
Looks good to me!

**Comment by vturpin** [comment]
@castelao who are the reviewers despite you and I? @soerenthomsen maybe, but he is gone now and should not take this responsibility anymore. How can we do that?

@soerenthomsen could you validate this PR in the meantime please ;)

**Comment by castelao** [comment]
@vturpin , good question. We don't have explicit reviewers for this repository. It's probably a good idea to think about and define it.

This is a special repository without a specific theme, but a support for the full organization. So it might make sense for a support team, or maybe the tasks team leaders? or both.

**Review by callumrollo — APPROVED** [review]

**Pull request merged**